pkgs/top-level/aliases.nix

```diff
@@ -1081,6 +1081,7 @@ mapAliases ({
   trezor_agent = trezor-agent; # Added 2024-01-07
   trustedGrub = throw "trustedGrub has been removed, because it is not maintained upstream anymore"; # Added 2023-05-10
   trustedGrub-for-HP = throw "trustedGrub-for-HP has been removed, because it is not maintained upstream anymore"; # Added 2023-05-10
+  turbogit = throw "turbogit has been removed, as it depends on old version of libgit2 that got removed"; # Added 2024-02-11
```

Comment:
I'd split this change out and discuss it separately.

Reply:
The [4.0.0](https://github.com/b4nst/turbogit/releases/tag/v4.0.0) release was over 1.5 years ago and the [5.0.0](https://github.com/b4nst/turbogit/releases/tag/v5.0.0) was almost a year ago. The package also didn't [receive much activity since December 2021](https://github.com/NixOS/nixpkgs/issues?q=sort%3Aupdated-desc+turbogit) except treewide changes like this one and could be considered a smaller burden for libraries like libgit2.

I don't think moving this to another PR will change anything about this and just prolong that we remove old libgit2 versions. We could just remove libgit2 and mark the package broken but that would also just delay the inevitable.

I tried updating the package to a newer version but that wasn't straight forward and I gave up after 40 minutes.

Reply:
That there is a newer release makes an even stronger case against removing it. Doesn't mean you have to update it, just that we have to wait longer for removing that one libgit2 version. It's probably affected by some CVE anyways, so stick an insecure warning into `meta` and we should be okay to move on until someone takes care of that.

Reply:
> That there is a newer release makes an even stronger case against removing it.

How that? It just proves that the package is unmaintained.

Reply:
I verified it still builds but only if you allow the insecure libgit2 package.